src/main/kotlin/com/terraformation/backend/report/render/ReportRenderer.kt

```diff
@@ -44,4 +48,83 @@ class ReportRenderer(
       }
     }
   }
+
+  fun renderReportCsv(reportId: ReportId): String {
+    val report = reportStore.fetchOneById(reportId)
+
+    val body: ReportBodyModel = report.body
+    val columnValues: List<Pair<String, Any?>> =
+        when (body) {
+          is ReportBodyModelV1 -> {
```

Comment:
return empty list otherwise?

Reply:
`ReportBodyModel` is a sealed interface, so it's a compile error if there are cases this `when` doesn't cover.

Reply:
Got it 👍